### DISCUSSION TITLE
你好！我打包的网站，提问回复导出word和pdf在macos下他会自动开打

### DISCUSSION THREAD

**Discussion by lqp518** [tw93/Pake#534] [Q&A]
这个是我打包的客户端https://github.com/lqp518/Pake/suites/14614121171/artifacts/828451082   请问怎么处理，要按command <- ->才能回去，普通用户不知道只能退出重新启动软件，windows的客户端就不会自动打开

**Comment by tw93** [comment]
应该马上就支持了，等这次dev合并到master就好，或者你可以用dev分支打包试试
